src/main/java/org/broadinstitute/hellbender/tools/copynumber/allelic/alleliccount/AllelicCountCollector.java

```diff
@@ -35,100 +25,44 @@
+    private final AllelicCountCollection allelicCounts = new AllelicCountCollection();
 
-    private final SamReaderFactory readerFactory;
-    private final ReferenceSequenceFileWalker referenceWalker;
-
-    /**
-     * Constructs a {@link AllelicCountCollector} object for calculating {@link AllelicCountCollection} objects from
-     * BAMs based on a reference genome.
-     * @param referenceFile         file containing the reference
-     * @param validationStringency  validation stringency to use for reading BAM files
-     */
-    public AllelicCountCollector(final File referenceFile,
-                                 final ValidationStringency validationStringency) {
-        IOUtils.canReadFile(referenceFile);
-        readerFactory = SamReaderFactory.makeDefault().validationStringency(validationStringency).referenceSequence(referenceFile);
-        referenceWalker = new ReferenceSequenceFileWalker(referenceFile);
+    public AllelicCountCollector() {
     }
 
-    /**
-     * Returns an {@link AllelicCountCollection} based on the pileup at sites (specified by an interval list)
-     * in a sorted BAM file.  Reads and bases below the specified mapping quality and base quality, respectively,
-     * are filtered out of the pileup.  The alt count is defined as the total count minus the ref count, and the
-     * alt nucleotide is defined as the non-ref base with the highest count, with ties broken by the order of the
-     * bases in {@link AllelicCountCollector#BASES}.
-     * @param bamFile           sorted BAM file
-     * @param siteIntervals     interval list of sites
-     * @param minMappingQuality minimum mapping quality required for reads to be included in pileup
-     * @param minBaseQuality    minimum base quality required for bases to be included in pileup
-     * @return                  AllelicCountCollection of ref/alt counts at sites in BAM file
-     */
-    public AllelicCountCollection collect(final File bamFile,
-                                          final IntervalList siteIntervals,
-                                          final int minMappingQuality,
-                                          final int minBaseQuality) {
-        try (final SamReader reader = readerFactory.open(bamFile)) {
-            ParamUtils.isPositiveOrZero(minMappingQuality, "Minimum mapping quality must be nonnegative.");
-            ParamUtils.isPositiveOrZero(minBaseQuality, "Minimum base quality must be nonnegative.");
-            if (reader.getFileHeader().getSortOrder() != SAMFileHeader.SortOrder.coordinate) {
-                throw new UserException.BadInput("BAM file " + bamFile.toString() + " must be coordinate sorted.");
-            }
-
-            final int numberOfSites = siteIntervals.size();
-            final boolean useIndex = numberOfSites < MAX_INTERVALS_FOR_INDEX;
-            final SamLocusIterator locusIterator = new SamLocusIterator(reader, siteIntervals, useIndex);
+    //TODO: Populate docs
```

Comment:
Don't forget this TODO.  Can probably just copy and paste some variation of the class javadoc.